src/DateRangePickerInput/web/DateRangePickerInput.js

```diff
@@ -224,23 +225,23 @@ class DateRangePickerInput extends React.Component {
       modifiers,
       renderDay,
       disabledDays,
-    } = this.props;
-
-    const {
       error,
     } = this.props;
 
     const {
       formik,
     } = this.context;
 
+    const errorMessage = cloneDeep(error);
```

Comment:
Do we really need to `clone` it? what if we create a new error object since we need this to just set error on fields.